bad1.in:2:type error
